docs/examples/tunnel-with-reverse-proxy/README.md

```diff
@@ -0,0 +1,161 @@
+  This configuration allows additional routes to be added with a tunnel binding to use this reverse proxy in combination with other routes.
+- You require reverse proxy functionality that cloudflared does not have
+- You want to keep your reverse proxy configuration modular, so that you can use any load balancer implementation
+- You use a service mesh with its own ingress implementation
+- You have multiple load balancers pointing at your ingress. For example a lab might include metallb + DNS entries so that you are not traversing the internet when on the same network as your cluster
+```
++----------------------+
+|   internet users     |
++----------------------+
+           |
+           v
++----------------------+
+|  cloudflare tunnel   |
++----------------------+
+           |
+           v
++----------------------+
+|    reverse proxy     |
++----------------------+
+     |           |
+     v           v
++---------+ +----------+
+| App #1  | |  App #2  |
++---------+ +----------+
+```
+
+## What have we got here
+
+In this example we will:
+1. Deploy cloudflare-operator and configure a cluster tunnel (this would also work with a regular tunnel resource).
+2. Deploy ingress-nginx as a reference reverse proxy, and configure the created cluster tunnel to point to ingress-nginx.
+3. Deploy an example application, and configure it behind nginx with an ingress resource
+   
+Doing so will allow you to use the cloudflare operator as a shim to whatever ingress method you choose.
+One can change the ingress implementation at will and use e.g. gateway API, a different ingress, etc
+
+Notes:
+- We use a cluster tunnel, but one can equally use a tunnel resource in the same way.
+- We do not use https with the reverse proxy for simplicity. 
+  The example deployment yaml includes commented snippets one can use to enable https.
+
+```
+.
+├── README.md                   # this document
+└── manifests                   # contains all manifests from this example
+    ├── cloudflare-operator     # contains manifests for configuring cloudflare-operator
+    │   ├── tunnel-binding.yaml    
+    │   └── cluster-tunnel.yaml
+    ├── hello                   # a reference application, you would replace this with your own application
+    │   ├── deployment.yaml
+    │   ├── ingress.yaml
+    │   └── service.yaml    
+    └── ingress-nginx           # this example deploys ingress-nginx as the reverse proxy
+        ├── kustomization.yaml
+        ├── namespace.yaml
+        └── values.yaml
+```
+
+## Steps
+
+> All steps assume you are working from the directory this README is in.
+
+> **Ensure that you have kubectl, kustomize, and helm installed.**
+
+1. Configure a secret with your API token or API key, as described in [the getting started guide](../../getting-started.md#cloudflare-tokens)
+2. Deploy the cloudflare operator
+```shell
+ kustomize build ../../../config/default | kubectl apply -f -
+```
```

Comment:
You could just link the getting started guide here, in case the deployment instructions change down the line.

Reply:
That's a good point, will do

Reply:
done